README.md

```diff
@@ -259,6 +259,11 @@ sda-commons library versions.
 The module [`sda-commons-client-jersey-example`](./sda-commons-client-jersey-example/README.md)
 presents an example application that shows how to invoke services.
 
+#### OIDC
+
+The module [`sda-commons-client-oidc`](./sda-commons-client-oidc/README.md) provides support for requesting OIDC access tokens
+from respective provides (e.g. Keycloak) for subsequent requests to other services.
+
```

Comment:
As the dependencies seem to be pretty similar it could be part of the client-jersey module in my opinion. That gives an opportunity for easier integration later. For example one could create an OIDC Client directly from the bundle or we can add the filter that is mentioned in the readme within the builder of the regular clients.

Reply:
This was the plan in my original PR. But back then it wasn’t clear whether the service account/technical user approach was the universal Solution and should be used everywhere.